catalog/ui/search-ui/standard/src/main/webapp/templates/search/searchForm.handlebars

```diff
@@ -208,7 +208,7 @@
 
                         <div class="input-group input-group-sm">
                             <span class="input-group-addon">Radius&nbsp;</span>
```

Comment:
See @andrewkfiedler comments here: 
https://github.com/codice/ddf/pull/214

I think removing the model binding now makes a lot more sense.